nautobot/docs/user-guide/platform-functionality/webhook.md

```diff
@@ -1,135 +1,119 @@
+If no body template is provided, Nautobot sends a default JSON payload:
 
-```no-highlight
+```json
 {
     "event": "created",
-    "timestamp": "2023-02-14 12:34:56.000000+00:00",
+    "timestamp": "2023-02-14T12:34:56.000000+00:00",
     "model": "location",
     "username": "admin",
     "request_id": "fab0a4fb-52ba-4cb4-9756-4e6a3ac05332",
-    "data": {
-        "id": "5e4f9a91-372b-46df-a50a-c26357475bee",
-        "display": "Campus A",
-        "url": "/api/dcim/locations/5e4f9a91-372b-46df-a50a-c26357475bee/",
-        "name": "Campus A",
-        "status": {
-            "display": "Active",
-            "id": "363a431c-c784-40b5-8513-758cafd174ad",
-            "url": "/api/extras/statuses/363a431c-c784-40b5-8513-758cafd174ad/",
-            "name": "Active",
-            "created": "2023-02-14T00:00:00Z",
-            "last_updated": "2023-02-14T19:40:13.216150Z"
-        },
-        ...
-    },
+    "data": { "id": "5e4f9a91-372b-46df-a50a-c26357475bee", "name": "Campus A" },
     "snapshots": {
         "prechange": null,
-        "postchange": {
-            "id": "5e4f9a91-372b-46df-a50a-c26357475bee",
-            "asn": null,
-            "url": "/api/dcim/locations/5e4f9a91-372b-46df-a50a-c26357475bee/",
-            "name": "Campus A",
-            ...
-        },
-        "differences": {
-            "removed": null,
-            "added": {
-                "id": "5e4f9a91-372b-46df-a50a-c26357475bee",
-                "asn": null,
-                "url": "/api/dcim/locations/5e4f9a91-372b-46df-a50a-c26357475bee/",
-                "name": "Campus A",
-                ...
-            }
-        }
+        "postchange": { "id": "5e4f9a91-372b-46df-a50a-c26357475bee", "name": "Campus A" },
+        "differences": { "added": { "id": "5e4f9a91-372b-46df-a50a-c26357475bee", "name": "Campus A" } }
     }
 }
 ```
 
 ## Webhook Processing
 
-When a change is detected, any resulting webhooks are placed into a Redis queue for processing. This allows the user's request to complete without needing to wait for the outgoing webhook(s) to be processed. The webhooks are then extracted from the queue by the `celery worker` process and HTTP requests are sent to their respective destinations.
+When Nautobot detects a relevant change, it queues the webhook in Redis. This ensures that webhook processing does not delay the original request. A Celery worker then dequeues and sends the HTTP request to the specified receiver.
 
-A request is considered successful if the response has a 2XX status code; otherwise, the request is marked as having failed. Failed requests may be retried manually via the admin UI.
+A webhook request is considered successful if the receiver responds with a `2XX` status code. Failed requests can be retried manually via the admin UI.
 
-## Troubleshooting
+## Troubleshooting Webhooks
```

Comment:
We need to change the link on line 845 in `nautobot-server.md` to match this header.

Reply:
Still needed.